### DISCUSSION TITLE
How to get length of an array?

### DISCUSSION THREAD

**Discussion by ArenaSu** [cue-lang/cue#1916] [Q&A] [answered]
I create a test.cue in which define Parameter as k8s object array (slice in golang), how can i get  length of Parameter?

**Comment by mvdan** [comment, marked as answer]
In a CUE config, you can use the `len` builtin: https://cuelang.org/docs/references/spec/#len

For a list (equivalent to a slice or array), it gives you:

> list              list length, smallest length for an open list

**Comment by iameggi** [comment]
u can use len
exemple ` len(arr)`

```
arr := [5]int{1, 2, 3, 4, 5}
	arrLength := len(arr)
	fmt.Println("Length of Array is :", arrLength)
```
